guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -0,0 +1,73 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 0
+
+title Installer-based provisioning with full-host boot disk on bare metal
+
+left header
+In the described scenario, Foreman does not manage DHCP.
+end header
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
+note over Host : Powered off
+
+== Create host in Foreman ==
+
+User -> Foreman : Create host
+User -> Foreman : Assign an MAC address
+Foreman -> Proxy : Create DNS records
+Proxy -> DNS : Forward DNS records
+Proxy -> Foreman : Report status of DNS
+Foreman -> User : Host is created\n(build mode enabled)
+
+User -> Foreman : Request to generate the full host boot-disk ISO
+group Templates [Boot disk iPXE - host, PXELinux, PXEGrub2]
+    Foreman -> Proxy : Render boot-loader configuration files
+end
+Foreman -> Proxy : Download kernel and initial RAM disk
+Foreman -> Foreman : Add the boot loader to ISO files
+Foreman -> Foreman : Build the full host boot-disk ISO
```

Comment:
```suggestion
Foreman -> Foreman : Add PXELinux & PXEGrub2 config files
Foreman -> Foreman : Build the full host boot-disk ISO
```

Reply:
Makes sense, however, elsewhere we've been calling these files "boot-loader configuration" -- is that ok?

Reply:
ACK